modules/spring-config/src/main/java/org/mule/runtime/config/internal/context/SpringMuleContextServiceConfigurator.java

```diff
@@ -247,7 +251,8 @@ public class SpringMuleContextServiceConfigurator extends AbstractSpringMuleCont
       .put(MULE_CORE_EXPORTER_FACTORY_KEY, getBeanDefinition(OpenTelemetrySpanExporterFactory.class))
       .put(MULE_CORE_EVENT_TRACER_KEY, getBeanDefinition(SelectableCoreEventTracer.class))
       .put(MULE_CORE_COMPONENT_TRACER_FACTORY_KEY, getBeanDefinition(CoreEventComponentTracerFactory.class))
-      .put(MULE_METER_PROVIDER_KEY, resolveMeterProvider())
+      .put(MULE_ERROR_METRICS_FACTORY_KEY, resolveErrorMetricsFactory())
+      .put(MULE_CONTAINER_METER_PROVIDER_KEY, resolveContainerMeterProvider())
```

Comment:
Is this used or injected in some context? It is placed in the mule context of an artifact.